packages/theme/src/index.tsx

```diff
@@ -31,6 +31,22 @@ export const SmashingThemeProvider = ({
   )
 }
 
+export const SmashingThemeModifier = ({
```

Comment:
Consistency: we're using `ThemeProvider` and `ThemeContext`, why not `ThemeModifier` instead of `SmashingThemeModifier`?

Reply:
`ThemeContext` and `ThemeProvider` is imported from styled components. `@smashing/theme` exports `SmashingThemeProvider` and `SmashingThemeModifier`. I guess that it's consistent

Reply:
Correct!